string_assignment.js

```diff
@@ -0,0 +1,194 @@
+//# String Exercises
+
+//1. console.log the first character of a string.
+console.log("1._________________________________");
+
+let string = "Hello";
+
+console.log(string[0]);
+
+
+//2. console.log the length of a string.
+console.log("2._________________________________");
+
+console.log(string.length);
+
+//3. console.log the last character of any string.
+
+console.log("3._________________________________");
+
+console.log(string[string.length - 1]);
+
+//4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+//
+
+console.log("4._________________________________");
+
+let capitalized = string.slice(0,(string.length - 1)) + string[string.length - 1].toUpperCase();
+
+console.log(capitalized);
+
+
+// 5. Create a drEvil code block that will take a single number varaible, and log the '<variablbeAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+//
+
+console.log("4._________________________________");
+
+let balance = 50;
```

Comment:
I think the question is simply asking to print out "1000000 dollars (pinky)" if amount is 1000000. Other than that your code looks fine.